docs/storage.md

```diff
@@ -0,0 +1,97 @@
+---
+title: Storage
+sort_rank: 5
+---
+
+# Storage
+
+Prometheus includes a local on-disk time series database, but also optionally integrates with remote storage systems.
+
+## Local storage
+
+Prometheus's local time series database stores time series data in a custom format on disk.
+
+### On-disk layout
+
+Ingested samples are grouped into blocks of two hours (by default). Each two-hour block consists of a directory containing one or more chunk files that contain all time series samples for that window of time, as well as a metadata file and index file (which indexes metric names and labels to series in the chunk files). The block for currently incoming samples is kept in memory and not fully persisted yet. It is secured against crashes by a write-ahead-log (WAL) that can be replayed when the Prometheus server restarts after a crash. When series are deleted via the API, deletion records are stored in separate tombstone files (instead of deleting the data immediately from the chunk files).
```

Comment:
I was under the impression that default was not something that should be changed, the flags are only for testing.

Reply:
Not sure, @fabxc to confirm. I thought under certain load situations it may make sense to tune this.

Reply:
Yes, but rarely and only as a thought experiment so far. Might not be advisably to encourage it before encountering a relevant use case

Reply:
Yeah, I don't think I'm encouraging it. I just didn't want to make absolute statements about things that aren't absolute, thus the "by default" :)

Reply:
We don't add that level of caveat everywhere with the 5m staleness, so I'd avoid it here too.

Reply:
Ok